src/services/microfrontend-service.ts

```diff
@@ -31,6 +31,37 @@ export class MicroFrontendService {
     this.addMicroFrontendDescriptor(mfe)
   }
 
+  public findMicroFrontend(mfeName: string): MicroFrontend {
+    const bundleDescriptor: BundleDescriptor =
+      this.bundleDescriptorService.getBundleDescriptor()
+    const { microfrontends } = bundleDescriptor
+
+    const mfe = microfrontends.find(({ name }) => name === mfeName)
+
+    if (!mfe) {
+      throw new CLIError(
+        `${mfeName} does not exist in the microfrontends section of the Bundle descriptor`
+      )
+    }
+
+    return mfe
+  }
+
+  public removeMicroFrontend(mfeName: string): void {
+    const bundleDescriptor: BundleDescriptor =
+      this.bundleDescriptorService.getBundleDescriptor()
+    const { microfrontends } = bundleDescriptor
+
+    const updatedBundleDescriptor: BundleDescriptor = {
+      ...bundleDescriptor,
+      microfrontends: microfrontends.filter(({ name }) => name !== mfeName),
+    }
+
+    this.bundleDescriptorService.writeBundleDescriptor(updatedBundleDescriptor)
```

Comment:
invert this in order to be sure folder is deleted before updating the desc

Reply:
done